# -*- coding: mbcs -*-
#
# Abaqus/CAE Release 2018 replay file
# Internal Version: 2017_11_07-17.21.41 127140
# Run by am2548 on Fri Jan 29 15:10:33 2021
#

# from driverUtils import executeOnCaeGraphicsStartup
# executeOnCaeGraphicsStartup()
#: Executing "onCaeGraphicsStartup()" in the site directory ...
from abaqus import *
from abaqusConstants import *
session.Viewport(name='Viewport: 1', origin=(1.38281, 1.38426), width=203.55, 
    height=137.319)
session.viewports['Viewport: 1'].makeCurrent()
from driverUtils import executeOnCaeStartup
executeOnCaeStartup()
execfile('new_solver_2.py', __main__.__dict__)
#: C:\Users\am2548\TissueModel\Documents\PhD\Code\Data_1\Com_Vor_GN\Jan-29-2021_0002
#: Warning: Index 0 in the sequence is out of range
#* Error: The mask results in an empty sequence
#* File "new_solver_2.py", line 242, in <module>
#*     define_material(network)
#* File "new_solver_2.py", line 94, in define_material
#*     edges=myModel.parts[partname].edges.getSequenceFromMask(mask=('[#1 ]', 
#* ), )),
